### ISSUE TITLE
distribute as ECMAScript module (ESM)

### ISSUE THREAD

**Issue by terencehonles** [rollbar/rollbar.js#1041]
I am using rollbar.js with Angular and by not providing an ESM distribution Angular complains this disables potential optimizations that ESM would allow. I am not sure if this is currently on the roadmap, but this is just a friendly request for this. I haven't looked at the code to see how much work this might be, but I figured I might as well make sure this was on the radar.

This is the specific warning and I am aware I can disable it https://angular.io/guide/build#configuring-commonjs-dependencies

**Comment by daguttt** [comment]
Yeah, this would be so nice. CommonJS makes our Angular bundles larger
